cms/src/components/groups/GroupSelector.spec.ts

```diff
@@ -0,0 +1,173 @@
+import "fake-indexeddb/auto";
+import { describe, it, expect, beforeEach, afterEach, vi, beforeAll, afterAll } from "vitest";
+import { mount } from "@vue/test-utils";
+import {
+    mockGroupDtoPrivateContent,
+    mockGroupDtoPublicContent,
+    mockGroupDtoPublicEditors,
+    mockGroupDtoPublicUsers,
+    mockGroupDtoSuperAdmins,
+    superAdminAccessMap,
+} from "@/tests/mockdata";
+
+import GroupSelector from "./GroupSelector.vue";
+import { Combobox } from "@headlessui/vue";
+import LTag from "../content/LTag.vue";
+import { accessMap, AclPermission, db } from "luminary-shared";
+import waitForExpect from "wait-for-expect";
+import { access } from "fs";
+
+describe("GroupSelector", () => {
+    // Need to set the access map before starting the tests. When moving this to beforeAll, it fails for some or other reason.
+    accessMap.value = superAdminAccessMap;
```

Comment:
Probably because you're modifying the result of an import. Not necessarily wrong in tests, but I suspect the change is only locally scoped in the `beforeAll`, and therefore not propagating to any other methods, while doing it here changes it for the whole scope, including the test methods.